images/nfs/Makefile

```diff
@@ -40,13 +42,26 @@ IMAGE_FILENAME := $(IMAGE_OUTPUT_DIR)/nfs.tar.gz
 
 do.build:
 	@echo === docker build $(NFS_IMAGE)
-	@cp Dockerfile $(TEMP)
+	@if [ $(GOARCH) == arm64 ] && [ -z `which apt-get` ]; then \
+		echo === Skip cross build for Rook NFS; \
+		exit 0; \
+	fi
+	@if [ $(GOARCH) == arm64 ]; then \
+		apt-get install -yq --no-install-recommends binfmt-support qemu-user-static; \
+		docker run --rm --privileged multiarch/qemu-user-static:register --reset; \
+		docker run --rm --privileged multiarch/qemu-user-static:register; \
+		sed -e '/FROM NFS_BASEIMAGE/r Dockerfile_header_arm64' Dockerfile > ${DOCKERFILE}; \
+		cd $(TEMP); \
+		curl https://github.com/multiarch/qemu-user-static/releases/download/v3.0.0/qemu-aarch64-static -o qemu-aarch64-static; \
+		chmod a+x qemu-aarch64-static; \
```

Comment:
We have instructions in the development documentation for the qemu environment setup.
Also it seems bad practice to me to let a Makefile run `apt-get install`.